Util/readMappings.m

```diff
@@ -0,0 +1,49 @@
+function [mappings] = readMappings(file, delimiter)
+% function [mappings] = readMappings(file, delimiter)
+%
+% This read a mapping file, with a predefined delimiter (',').
+%
+% Inputs:
+%
+% file - a file location string
+% delimiter - a field delimiter
+%             default: ','
+%
+% Outputs:
+%
+% mappings - a containers.Map mapping
+%
+% Example:
+%
+% file =
+% [mappings] = readMappings(file)
+% assert()
```

Comment:
would be good to write the assert. also, no need for a unittest?

Reply:
Yes, Good catch. I fixed some aspects of the func days ago because a bug bite me just because of that missing docstring unittest.

I forgot to push the commit here though. See new commit below.